build_tools/bazel/build_bindings.sh

```diff
@@ -43,7 +43,6 @@ fi
 declare -a test_env_args=(
   --test_env=IREE_LLVMJIT_DISABLE=$IREE_LLVMJIT_DISABLE
   --test_env=IREE_VULKAN_DISABLE=$IREE_VULKAN_DISABLE
-  --action_env=IREE_LLVMAOT_LINKER_PATH=$IREE_LLVMAOT_LINKER_PATH
```

Comment:
How does this behave when the env var isn't set? We probably still want to forward the env var into Bazel's sandbox _if it is set explicitly_.

Reply:
Haven't tried `bazel` hopefully it will be null. I will revert this and keep the forwarding.